platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -101,7 +120,7 @@ typedef void (^MGLMapSnapshotCompletionHandler)(NSImage* _Nullable snapshot, NSE
  options.zoomLevel = 10
  
  var snapshotter = MGLMapSnapshotter(options: options)
- snapshotter.start { (image, error) in
+ snapshotter.start { (snapshot, error) in
```

Comment:
Add this example to MGLDocumentationExampleTests.swift, per [these instructions](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/DEVELOPING.md#adding-a-code-example) for adding a code example. Otherwise, the next time someone generates runtime styling code, this block may get overwritten.

Reply:
Was missed when the sample was added apparently. Added it now and updated the code.